### PR TITLE
Add initial .jscsrc

### PR DESCRIPTION
**Pull request by hzoo**
~~Only rule added was `disallowMultipleVarDecl`~~

This is just moving the existing rules. Should this be 1.0.0?

### CODE REVIEW

File: jscs.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  "disallowImplicitTypeConversion": [
+    "string"
+  ],
+  "disallowKeywords": [
+    "with"
+  ],
+  "disallowMixedSpacesAndTabs": true,
+  "disallowMultipleLineBreaks": true,
+  "disallowMultipleLineStrings": true,
+  "disallowMultipleVarDecl": true,
```
Comment (mikesherov) [inline jscs.json]: remove for now.

**Comment by mikesherov** [comment]
:+1:

**Comment by mikesherov** [comment]
Make this 1.0.0

**Comment by Aintaer** [comment]
:+1:

**Pull request merged**